fedimint-core/Cargo.toml

```diff
@@ -50,10 +50,12 @@ macro_rules_attribute = "0.2.0"
 miniscript = { workspace = true, features = ["serde"] }
 parity-scale-codec = { version = "3.7.4", features = ["derive"] }
 rand = { workspace = true }
+scopeguard = { workspace = true }
```

Comment:
I'm too lazy to define struct with drop just for basically a single line of code. 316M downloads on crates.io.